charts/istioconfig/values.yaml

```diff
@@ -0,0 +1,5 @@
+namespaces:
+  istio: istio-system
+
+gateway:
+  name: template-gateway
```

Comment:
I don't think either of these values are ever used in the chart, why are they here?